src/main/java/teammates/common/datatransfer/StudentAttributes.java

```diff
@@ -130,7 +129,7 @@ public boolean isRegistered() {
     }
 
     public String getRegistrationUrl() {
-        return new Url(Config.APP_URL + Const.ActionURIs.STUDENT_COURSE_JOIN_NEW)
+        return new Url(Const.ActionURIs.STUDENT_COURSE_JOIN_NEW)
```

Comment:
It's already done. The extensions in `StudentFeedbackResultsPageUiTest` are testing the redirects.